rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -173,6 +173,9 @@ public class Bridge extends PrecompiledContracts.PrecompiledContract {
     public static final CallTransaction.Function GET_PENDING_FEDERATOR_PUBLIC_KEY = BridgeMethods.GET_PENDING_FEDERATOR_PUBLIC_KEY.getFunction();
     // Returns the public key of given type the federator at the specified index for the current pending federation
     public static final CallTransaction.Function GET_PENDING_FEDERATOR_PUBLIC_KEY_OF_TYPE = BridgeMethods.GET_PENDING_FEDERATOR_PUBLIC_KEY_OF_TYPE.getFunction();
+  
+    // Returns the proposed federation bitcoin address
+    public static final CallTransaction.Function GET_PROPOSED_FEDERATION_ADDRESS = BridgeMethods.GET_PROPOSED_FEDERATION_ADDRESS.getFunction();
```

Comment:
Do you need this here? Who's using it?

Reply:
We don't need it here, but it seems there is a pattern to add this here. Let me know if we want to delete it.

Reply:
Please remove yes

Reply:
updated